imap/http_dav_sharing.c

```diff
@@ -381,44 +381,35 @@ static int _create_notify_collection(const char *userid, struct mailbox **mailbo
 
         r = mboxlist_createmailbox(mbentry, 0/*options*/, 0/*highestmodseq*/,
                                    1/*isadmin*/, userid, NULL/*authstate*/,
-                                   0/*flags*/, mailbox);
-        /* we lost the race, that's OK */
-        if (r == IMAP_MAILBOX_LOCKED) r = 0;
-        if (r) syslog(LOG_ERR, "IOERROR: failed to create %s (%s)",
+                                   0/*flags*/, NULL);
+
+        if (r) xsyslog(LOG_ERR, "IOERROR: failed to create notify collection"
+		      "mailbox=<%s> error=<%s>",
```

Comment:
This xsyslog call is also missing the comma between description string and format string

```suggestion
        if (r) xsyslog(LOG_ERR, "IOERROR: failed to create notify collection",
		      "mailbox=<%s> error=<%s>",
```